### ISSUE TITLE
Can't build on Mac (ocaml-lsp-server - Library "cmdliner" not found)

### ISSUE THREAD

**Issue by robmurtagh** [bryphe/revery-quick-start#70]
Hi, I can't build revery-quick-start on Mac OSX Catalina 10.15.3, `esy build` fails with error:

```txt
error: build failed with exit code: 1
  build log:
    # esy-build-package: building: @opam/ocaml-lsp-server@github:ocaml/ocaml-lsp:ocaml-lsp-server.opam#04733ed
    # esy-build-package: pwd: /Users/robmurtagh/.esy/3______________________________________________________________/b/opam__s__ocaml_lsp_server-ba097269
    # esy-build-package: running: 'dune' 'build' '-j' '4' '--root' '.' '--ignore-promoted-rules' '--no-config' '--profile' 'release' 'ocaml-lsp-server.install'
    File "ocaml-lsp-server/src/dune", line 8, characters 3-11:
    8 |    cmdliner))
           ^^^^^^^^
    Error: Library "cmdliner" not found.
    Hint: try:
    dune external-lib-deps --missing --no-config --root . --ignore-promoted-rules
    --profile release -j 4 ocaml-lsp-server.install
    error: command failed: 'dune' 'build' '-j' '4' '--root' '.' '--ignore-promoted-rules' '--no-config' '--profile' 'release' 'ocaml-lsp-server.install' (exited with 1)
    esy-build-package: exiting with errors above...
    
  building @opam/ocaml-lsp-server@github:ocaml/ocaml-lsp:ocaml-lsp-server.opam#04733ed
esy: exiting due to errors above
```

I can fix it by rolling back to a previous version before [this diff](https://github.com/revery-ui/revery-quick-start/commit/918dd49d04cc17065e1c2fdfb8de964dfa08ccba#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R42). e.g.

```sh
git checkout a99fee9dce8deea6d84a392994cf68d01a348711
esy build && esy run
```

Any idea of the best way to resolve this one?

Thanks in advance!

**Comment by glennsl** [comment]
I can't reproduce this, either with the lockfiles or after removing them, although the latter surfaced other issues. Those issues are fixed in #73. Maybe that'll fix this too. Could you try that @robmurtagh ?

**Comment by robmurtagh** [comment]
https://github.com/revery-ui/revery-quick-start/pull/73 fixes this issue for me @glennsl, thanks 🙌
